domains/POAS/events/immaterial-change-confirmed/schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$id": "https://opg.service.justice.gov.uk/opg.poas.lpastore/immaterial-change-confirmed.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "opg.poas.lpastore/immaterial-change-confirmed",
+  "type": "object",
+  "properties": {
+    "uid": {
+      "type": "string",
+      "description": "The UID of the LPA",
+      "pattern": "^M(-[A-Z0-9]{4}){3}$"
+    },
+    "actorUID": {
+      "type": "string",
+      "description": "The UID of the actor that made the change",
+      "pattern": "^([a-z0-9]{8}-)([a-z0-9]{4}-){3}([a-z0-9]{12})$"
+    },
+    "actorType": {
+      "type": "string",
+      "description": "The type of actor that made the change",
+      "enum": ["donor", "certificateProvider", "attorney"]
+    }
+  },
+  "if": {
+    "properties": {
+      "actorType": { "const": "attorney" }
+    }
+  },
```

Comment:
This is in preparation for tickets like [this](https://opgtransform.atlassian.net/browse/MLPAB-2585) but happy to drop if we'd rather keep this targetted at donors and CPs for now.

Reply:
I feel like it might be simpler to always have `actorUID`, even if we don't really need it